frontend/src/pages/admin/questions/update/[id].tsx

```diff
@@ -128,7 +131,7 @@ const QuestionUpdate: NextPage = (): JSX.Element => {
                   disabled={isSubmitting}
                   {...register('category_id', { required: 'Category is required' })}
                 >
-                  <option value={category?.id}>{category?.title}</option>
+                  <option defaultValue={category?.id}>{category?.title}</option>
```

Comment:
I think no need na to check if value exists kay its already set to sa `useState`
```suggestion
                  <option defaultValue={category.id}>{category.title}</option>
```